pex/commands/bdist_pex.py

```diff
@@ -67,6 +68,10 @@ def split_and_strip(entry_point):
           for script in self.distribution.entry_points.get('console_scripts', []))
     except ValueError:
       console_scripts = {}
+    except AttributeError:
```

Comment:
This is a bit cryptic / indirect - where is `AttributeError` raised from - takes a bit of staring.  How about being direct and restructuring to test `self.distribution.entry_points` with:
```python
if isinstance(self.distribution.entry_points, Compatibility.string): # See: https://github.com/pantsbuild/pex/blob/master/pex/compatibility.py#L35
  # ConfigParser
elif isinstance(self.distribution.entry_points, dict):
  # dict access
else:
  raise TypeError(...
```